content/pt-br/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -0,0 +1,125 @@
+  - Fedora CoreOS
+  - openSUSE Leap 15.x/Tumbleweed
+  - Oracle Linux 7, 8, 9
+  - Alma Linux 8, 9
+  - Rocky Linux 8, 9
+  - Kylin Linux Advanced Server V10
+  - Amazon Linux 2
+* Testes de integração contínua.
+
+Para escolher a ferramenta que melhor se adapta ao seu caso, leia [esta comparação](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/comparisons.md) com
+[kubeadm](/docs/reference/setup-tools/kubeadm/) e [kops](/docs/setup/production-environment/tools/kops/).
+
+<!-- body -->
+
+## Criando um cluster
+
+### (1/5) Atenda aos requisitos básicos
+
+Provisione servidores com os seguintes [requisitos](https://github.com/kubernetes-sigs/kubespray#requirements):
+
+* **A versão mínima exigida do Kubernetes é v1.22**
+* **Ansible v2.11+, Jinja 2.11+ and python-netaddr estão instalados na máquina que executará os comandos do Ansible**
+* Os servidores de destino devem ter acesso à Internet para extrair imagens do docker. Caso contrário, é necessária configuração adicional. Consulte ([Offline Environment](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/offline-environment.md))
+* Os servidores de destino são configurados para permitir o **encaminhamento IPv4**.
+* Se estiver usando IPv6 para pods e serviços, os servidores de destino serão configurados para permitir o **encaminhamento de IPv6**.
+* Os **firewalls não são gerenciados**, você precisará implementar suas próprias regras da maneira que costumava fazer. Para evitar qualquer problema durante o deployment, você deve desabilitar seu firewall.
+* Se o kubespray for executado a partir de uma conta de usuário não raiz, o método correto de escalonamento de privilégios deve ser configurado nos servidores de destino. Em seguida, o `ansible_become` flag ou os parâmetros de comando `--become` ou `-b` devem ser especificados.
+
+Kubespray fornece as seguintes utilidades para ajudar a provisionar seu ambiente:
+
+* Scripts do [Terraform](https://www.terraform.io/) para os seguintes provedores de nuvem:
+  * [AWS](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/aws)
+  * [OpenStack](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/openstack)
+  * [Equinix Metal](https://github.com/kubernetes-sigs/kubespray/tree/master/contrib/terraform/metal)
+
+### (2/5) Componha um arquivo de inventário
+
+Depois de provisionar seus servidores, crie um [arquivo de inventário para o Ansible](https://docs.ansible.com/ansible/latest/network/getting_started/first_inventory.html). Você pode fazer isso manualmente ou por meio de um script de inventário dinâmico. Para mais informações, consulte "[Criando seu próprio inventário](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#building-your-own-inventory)".
+
+### (3/5) Planeje o deployment do seu cluster
+
+O Kubespray oferece a habilidade de customizar vários aspectos do deployment:
+
+* Escolha do modo de deployment: kubeadm ou non-kubeadm
```

Comment:
Perhaps use "criação" instead of "deployment" here? In this context, deployment is not referring to the Kubernetes object, so I think it would be better to translate it.